docs/community/improvement-proposals.md

```diff
@@ -0,0 +1,74 @@
+# Kyuubi Project Improvement Proposals (KBIP)
+
+The purpose of an KBIP is to inform and involve the user community in major improvements to the Kyuubi codebase throughout the development process, to increase the likelihood that user needs are met.
```

Comment:
The purpose of a KBIP is to inform and involve the user community in significant improvements to the Kyuubi codebase throughout the development process to increase user needs.